Source/Example Measurements/NI-DCPower Source DC Voltage/NI-DCPower Measurement/NIDCPowerSourceDCVoltage.lvlib

```diff
@@ -0,0 +1,34 @@
+
+</Property>
+	<Item Name="Advanced" Type="Folder">
+		<Item Name="Build Assets" Type="Folder">
+			<Item Name="Post-Build Action.vi" Type="VI" URL="../Advanced/Build Assets/Post-Build Action.vi"/>
+		</Item>
+		<Item Name="MeasurementPlugin.lvclass" Type="LVClass" URL="../Advanced/MeasurementPlugin.lvclass"/>
+	</Item>
+	<Item Name="subVIs" Type="Folder">
+		<Item Name="Convert Location to Address String.vi" Type="VI" URL="../subVIs/Convert Location to Address String.vi"/>
+		<Item Name="DCPower Specific Logic.vi" Type="VI" URL="../subVIs/DCPower Specific Logic.vi"/>
+		<Item Name="Location to Grpc Device.vi" Type="VI" URL="../subVIs/Location to Grpc Device.vi"/>
+	</Item>
+	<Item Name="TestStand Code Modules" Type="Folder">
+		<Item Name="Create and Register Sessions.vi" Type="VI" URL="../TestStand Code Modules/Create and Register Sessions.vi"/>
+		<Item Name="Unregister and Destroy Sessions.vi" Type="VI" URL="../TestStand Code Modules/Unregister and Destroy Sessions.vi"/>
```

Comment:
Do not pass timeout=0 to ReserveSessions and ReserveAllRegisteredSessions. Either choose a timeout or pass timeout=-1 for infinite. The Python examples pass timeout=-1.

This also applies to Measurement Logic.vi.

Reply:
I talked to Jonathan Mendez about this. He didn't make a recommendation, but told me that a 0 timeout means that we will try to reserve a session and if it is already reserved it will immediately throw an error. That seems reasonable to me, especially in the code module where we expect the sessions to be available. I would argue that -1 is the worst behavior because it will cause your application to hang and you won't know why. If my understanding is incorrect, let me know.

Reply:
Ok, how about 60000 (1 min) for the measurement and 0 for the code module? We should do the same in Python.

Reply:
Sounds reasonable to me. I while file an issue to python.